Command/CleanMediaCommand.php

```diff
@@ -25,11 +25,6 @@
 class CleanMediaCommand extends ContainerAwareCommand
 {
     /**
-     * @var MediaProviderInterface[]|false
-     */
-    private $providers = false;
```

Comment:
I changed a little bit the description, shall I move that description to the commit message body?